data/json/itemgroups/locations.json

```diff
@@ -83,6 +83,13 @@
       ["material_niter", 20],
       ["material_limestone", 5]
     ]
+  },{
+    "type":"item_group",
+    "id":"prison_textile",
+    "subtype": "collection",
+    "entries" : [
+      { "item": "blanket", "prob": 25 },
+      { "item": "jumpsuit", "prob": 35 }
```

Comment:
Jumpsuits were in original mapgen - see https://github.com/CleverRaven/Cataclysm-DDA/blob/master/src/mapgen.cpp#L7354.

I believe we better change loot once all mapgen is moved to json from cpp.